src/cromshell/utilities/cromshellconfig.py

```diff
@@ -173,6 +177,11 @@ def __get_cromwell_server(config_options: dict):
     return config_options["cromwell_server"]
 
 
+def get_cromwell_api():
```

Comment:
Change this from string to function so that if the cromwell server gets updated after comparing workflow id with submission file, cromwell_api provides the latest cromwell server